modulemd/v2/include/modulemd-2.0/modulemd-rpm-map-entry.h

```diff
@@ -0,0 +1,230 @@
+/*
+ * This file is part of libmodulemd
+ * Copyright (C) 2018 Red Hat, Inc.
+ *
+ * Fedora-License-Identifier: MIT
+ * SPDX-2.0-License-Identifier: MIT
+ * SPDX-3.0-License-Identifier: MIT
+ *
+ * This program is free software.
+ * For more information on the license, see COPYING.
+ * For more information on free software, see <https://www.gnu.org/philosophy/free-sw.en.html>.
+ */
+
+#pragma once
+
+#include <glib-object.h>
+
+#include "private/modulemd-util.h"
+
+G_BEGIN_DECLS
+
+/**
+ * SECTION: modulemd-rpm-map-entry
+ * @title: Modulemd.RpmMapEntry
+ * @stability: stable
+ * @short_description: Provides an exploded and unexploded view of the NEVRA of
+ * an RPM artifact.
+ */
+
+#define MODULEMD_TYPE_RPM_MAP_ENTRY (modulemd_rpm_map_entry_get_type ())
+
+G_DECLARE_FINAL_TYPE (ModulemdRpmMapEntry,
+                      modulemd_rpm_map_entry,
+                      MODULEMD,
+                      RPM_MAP_ENTRY,
+                      GObject)
+
+/**
+ * modulemd_rpm_map_entry_new:
+ * @name: The name of the package
+ * @epoch: The epoch of the package
+ * @version: The version of the package
+ * @release: The release string of the package
+ * @arch: The processor architecture of the package
+ *
+ * Create a new rpm-map entry. This will contain the decomposed NEVRA of an
+ * RPM artifact.
+ *
+ * Returns: (transfer full): A newly-allocated #ModulemdRpmMapEntry with the
+ * provided values.
+ */
+ModulemdRpmMapEntry *
+modulemd_rpm_map_entry_new (const gchar *name,
+                            guint32 epoch,
```

Comment:
Why isn't this a const as well?

Reply:
The epoch value? The `const` would be meaningless here as the value is copied, not passed by reference.